### PR TITLE
Logging doc to link to ASP.NET Core logging

### PR DESCRIPTION
**Pull request by cijothomas**
I'll add PR comment with some reasoning for this, as well as some concerns..

### CODE REVIEW

File: docs/logs/getting-started/README.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ instance, which is then used to do the logging. The log is sent to the
 `OpenTelemetryLoggerProvider`, which is configured to export logs to
 `ConsoleExporter`. `ConsoleExporter` simply displays it on the console.
 
+If you are using ASP.NET Core applications, then adding logging providers
```
Comment (cijothomas) [inline docs/logs/getting-started/README.md:64]: I have seen users trying to create LoggerFactory by hand, and managing its lifecycle themselves (often incorrectly). Linking to the common workload type to advice users about this..

Ideally this should be covered fully in the .net logging docs, and we should simply link from opentelemetry to there. But as of today, the .NET logging docs for console apps (https://docs.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#non-host-console-app) is very minimal..

Comment (reyang) [inline docs/logs/getting-started/README.md:64]: Consider something like:

In a real application, the `ILogger` instance might be obtained using a very different way, here are some useful links:
* Logging in ASP.NET Core

Comment (reyang) [reply docs/logs/getting-started/README.md:64]: And maybe link to the example, e.g. https://github.com/open-telemetry/opentelemetry-dotnet/blob/6b0883eb224fb143b14e527cc622eeb584c58770/examples/AspNetCore/Program.cs#L89

Comment (cijothomas) [reply docs/logs/getting-started/README.md:64]: this sounds better.! let me tweak further.

File: docs/logs/getting-started/README.md [hunk 2]
```diff
@@ -61,6 +61,16 @@ instance, which is then used to do the logging. The log is sent to the
 `OpenTelemetryLoggerProvider`, which is configured to export logs to
 `ConsoleExporter`. `ConsoleExporter` simply displays it on the console.
 
+Adding logging providers, obtaining `ILogger` instance, etc. might be done
+differently based on the type of the application. Check the respective
```
Comment (reyang) [inline docs/logs/getting-started/README.md:65]: "check the respective documents" seems to indicate that this doc promises to provide/maintain a list of such documents, which is not the case?

File: docs/logs/getting-started/README.md [hunk 3]
```diff
@@ -61,6 +61,16 @@ instance, which is then used to do the logging. The log is sent to the
...
+documentations to learn about the idiomatic way of doing logs for each
+application type.
+
+* [Logging in ASP.NET
+Core](https://docs.microsoft.com/aspnet/core/fundamentals/logging#logging-providers)
+([Example](../../../examples/AspNetCore/Program.cs))
+
```
Comment (reyang) [inline docs/logs/getting-started/README.md:72]: nit: remove the extra white line

File: docs/logs/getting-started/README.md [hunk 4]
```diff
@@ -61,6 +61,15 @@ instance, which is then used to do the logging. The log is sent to the
 `OpenTelemetryLoggerProvider`, which is configured to export logs to
 `ConsoleExporter`. `ConsoleExporter` simply displays it on the console.
 
+Adding logging providers, obtaining `ILogger` instance, etc. could alternatively
+be done in ways, more idiomatic to the application type. For e.g. [Logging in
```
Comment (reyang) [inline docs/logs/getting-started/README.md:65]: This line sounds weird to me. Consider getting some help from native English speakers (I am not).

File: docs/logs/getting-started/README.md [hunk 5]
```diff
@@ -61,6 +61,13 @@ instance, which is then used to do the logging. The log is sent to the
 `OpenTelemetryLoggerProvider`, which is configured to export logs to
 `ConsoleExporter`. `ConsoleExporter` simply displays it on the console.
 
+Adding logging providers, obtaining `ILogger` instance, etc. could be done
+differently based on the application type. For e.g. [Logging in ASP.NET
```
Comment (alanwest) [inline docs/logs/getting-started/README.md:65]:
```suggestion
differently based on the application type. For example, [Logging in ASP.NET
```

**Review by reyang — APPROVED** [review]

**Review by alanwest — APPROVED** [review]

**Pull request merged**